test-collateral/generate-policy/src/main.rs

```diff
@@ -347,6 +353,18 @@ binary.",
                 .help("The configuration of the standard error in the form 'path:rights'")
                 .required(false),
         )
+        .arg(
+            Arg::with_name("enable-clock")
+                .short("c")
+                .long("enable-clock")
+                .help(
+                    "Specifies whether the Veracruz trusted runtime should allow the WASM \
+binary to call clock functions (`clock_getres()`, `clock_gettime()`).",
+                )
+                .required(true)
+                .value_name("BOOLEAN")
+                .default_value(&default_enable_clock),
```

Comment:
Don't forget to remove `required` and `default_value` here!

Reply:
Thanks @egrimley-arm . git rebase didn't spot that